recipes/coprarna/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "coprarna" %}
+{% set version = "alpha2" %}
+{% set md5 = "95abc9567e995331c6412f9b2f3ac6aa" %} 
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.zip
+  url: https://github.com/eggzilla/CopraRNA/archive/{{ version }}.zip
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  #  [osx]
+
+# commented versions are tested as working
+requirements:
+  build:
+    - perl >=5.22
+    - perl-bioperl >=1.6.924
+    - perl-list-moreutils ==0.413
+    - perl-parallel-forkmanager ==1.17
+    - perl-getopt-long
+    - perl-bio-eutilities ==1.75
+    - r-base >=3.2.2
+    - r-seqinr
+    - python ==2.7.13
```

Comment:
python should not be pinned like this, please skip python 3 in the build section